tensorflow2/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM python:3.7-slim
 COPY ./tensorflow2 /app
 WORKDIR /app
+RUN pip install --upgrade pip
```

Comment:
Please add `--no-cache-dir` to reduce the docker image size when executing pip install